src/patternfly/components/Wizard/wizard.scss

```diff
@@ -0,0 +1,419 @@
+.pf-c-wizard {
+  --pf-c-wizard--GridTemplateRows: auto auto 1fr auto;
+  --pf-c-wizard--md--GridTemplateRows: auto 1fr auto;
+  --pf-c-wizard--md--GridTemplateColumns: 300px 1fr;
+  --pf-c-wizard--Width: calc(100% - var(--pf-global--spacer--xl) * 2);
+  --pf-c-wizard--Height: calc(100% - var(--pf-global--spacer--xl) * 2);
+  --pf-c-wizard--BoxShadow: var(--pf-global--BoxShadow--lg);
+  --pf-c-wizard--children--BackgroundColor: var(--pf-global--BackgroundColor--100);
+
+  @media screen and (min-width: $pf-global--breakpoint--md) {
+    --pf-c-wizard--GridTemplateRows: var(--pf-c-wizard--md--GridTemplateRows);
+  }
+
+  // Header
+  --pf-c-wizard__header--PaddingTop: var(--pf-global--spacer--lg);
+  --pf-c-wizard__header--PaddingRight: var(--pf-global--spacer--xl);
+  --pf-c-wizard__header--PaddingBottom: var(--pf-global--spacer--lg);
+  --pf-c-wizard__header--PaddingLeft: var(--pf-global--spacer--xl);
+
+  // Close
+  --pf-c-wizard__close--Top: calc(var(--pf-global--spacer--lg) - var(--pf-global--spacer--form-element));
+  --pf-c-wizard__close--Right: var(--pf-global--spacer--md);
+  --pf-c-wizard__close--FontSize: var(--pf-global--FontSize--xl);
+
+  // Title
+  --pf-c-wizard__title--PaddingRight: var(--pf-global--spacer--xl);
+
+  // Description
+  --pf-c-wizard__description--PaddingTop: var(--pf-global--spacer--sm);
+  --pf-c-wizard__description--Color: var(--pf-global--Color--400);
+
+  // Steps
+  --pf-c-wizard__steps--md--BoxShadow: var(--pf-global--BoxShadow--md-right);
```

Comment:
does the `md` refer to the box shadow size?

Reply:
it's because it's a var in the `md` media query: `@media screen and (min-width: $pf-global--breakpoint--md)`

Reply:
oh nice!